### PR TITLE
Add access to more talons

### PR DESCRIPTION
**Pull request by jpmii**
## Description

Add access to checkout's useOverview talon

## Related Issue

#1784

## Acceptance 

@awilcoxa @jimbo @zetlen

### CODE REVIEW

File: packages/peregrine/lib/targets/TalonWrapperConfig.js [hunk 1]
```diff
@@ -33,6 +33,52 @@ class TalonWrapperConfig {
              */
             useApp: new WrappableTalon(addTransforms, 'App/useApp')
         };
+        /**
+         * Provides access to the talon used in Venia's Checkout UI component.
+         * @instance
+         */
+        this.Checkout = {
```
Comment (sirugh) [inline packages/peregrine/lib/targets/TalonWrapperConfig.js:40]: These are legacy checkout components/talons. Should we indicate such in the property name?

For example, `this.LegacyCheckout`? Or are these property names meant to literally be the name of the path to the talon such as in `Checkout/useFlow`?

@zetlen I remember we had a discussion around naming but I don't remember this specific point.

File: packages/peregrine/lib/targets/TalonWrapperConfig.js [hunk 2]
```diff
@@ -33,6 +33,52 @@ class TalonWrapperConfig {
...
+        /**
+         * Provides access to the talon used in Venia's SignIn UI component.
+         * @instance
+         */
+        this.SignIn = {
+            /**
+             * useSignin {@link WrappableTalon} object for the `useSignin()` talon
+             * @type {WrappableTalon}
+             */
+            useSignin: new WrappableTalon(addTransforms, 'SignIn/useSignin')
```
Comment (sirugh) [inline packages/peregrine/lib/targets/TalonWrapperConfig.js:80]: Match casing - I believe this is `useSignIn`.

```suggestion
            useSignIn: new WrappableTalon(addTransforms, 'SignIn/useSignIn')
```

**Review by sirugh — CHANGES_REQUESTED** [review]
Please run `prettier` and rebase!

File: packages/peregrine/lib/targets/TalonWrapperConfig.js (hunk 1 above)
Comment (jimbo) [inline packages/peregrine/lib/targets/TalonWrapperConfig.js:40]: In the latest versions of Venia, the drawer checkout and its talons here are essentially deprecated, in favor of `CheckoutPage`. Could we target `CheckoutPage` talons instead?

**Review by jimbo — CHANGES_REQUESTED** [review]
Thanks for the straightforward contribution, and apologies for letting it slip through the cracks for a while. I think we'd prefer to expose the new checkout work rather than the old stuff, but otherwise I think we could move forward with this.

File: packages/peregrine/lib/targets/TalonWrapperConfig.js (hunk 1 above)
Comment (jpmii) [reply packages/peregrine/lib/targets/TalonWrapperConfig.js]: In the newest commit, I changed this to CheckoutPage. Thanks!

File: packages/peregrine/lib/targets/TalonWrapperConfig.js [hunk 3]
```diff
@@ -33,6 +33,50 @@ class TalonWrapperConfig {
...
+             * useCheckoutPage {@link WrappableTalon} object for the `useCheckoutPage()` talon
+             * @type {WrappableTalon}
+             */
+            useCheckoutPage: new WrappableTalon(addTransforms, 'CheckoutPage/useCheckoutPage')
+        };
+        /**
+         * Provides access to the talon used in Venia's MiniCart UI component.
+         * @instance
+         */
+        this.MiniCart = {
```
Comment (sirugh) [inline packages/peregrine/lib/targets/TalonWrapperConfig.js:51]: Since we're not wrapping legacy talons you can just remove this object.

Comment (jimbo) [reply packages/peregrine/lib/targets/TalonWrapperConfig.js:51]: @sirugh Isn't this the new minicart though?

**Review by jimbo — APPROVED** [review]
Thanks for updating it. 🙂

File: packages/peregrine/lib/targets/TalonWrapperConfig.js (hunk 1 above)
Comment (jpmii) [reply packages/peregrine/lib/targets/TalonWrapperConfig.js]: > Please run `prettier` and rebase!

The latest commit should have the `prettier` results

File: packages/peregrine/lib/targets/TalonWrapperConfig.js (hunk 3 above)
Comment (sirugh) [reply packages/peregrine/lib/targets/TalonWrapperConfig.js:54]: Doh - you're right @jimbo. My mistake. Forgot we moved the legacy one to `LegacyMiniCart` :P

**Review by sirugh — APPROVED** [review]

**Review by larsroettig — APPROVED** [review]

**Comment by tjwiebell** [comment]
Tracking internally with [PWA-956](https://jira.corp.magento.com/browse/PWA-956). Sending to QA.

**Pull request merged**